Dockerfile

```diff
@@ -1,8 +1,10 @@
 # Build stage for compiling Zandronum
-FROM ubuntu:19.10 AS build
+FROM ubuntu:20.04 AS build
 WORKDIR /build
+ENV DEBIAN_FRONTEND=noninteractive
 RUN true \
     && apt-get update -qq \
+    && apt-get install apt-utils \
```

Comment:
Does this have to be installed prior to the other packages? If not, I'd rather have it in the list starting at line 9 (above `ca-certificates`) that way we're not running multiple `install` commands.

Reply:
Actually, the warning about apt-utils can be ignored. I removed its installation from my branch.